src/test/java/BaseTest.java

```diff
@@ -1,11 +1,183 @@
+                chromeOptions.addArguments("--incognito");
+                chromeOptions.addArguments("--start-maximized");
+                chromeOptions.setExperimentalOption("excludeSwitches", new String[]{"enable-automation"});
+                return driver = new ChromeDriver(chromeOptions);
+        }
+    }
+
+    public static WebDriver lambdaTest() throws MalformedURLException{
+        String hubURL ="http://hub.lambdatest.com/wd/hub";
+        String userName ="a23ska";
+        String accessKey ="51pxhdIT5SNNtUNX3dbZAeYq5wfT2JMoAoPKTS1MguEsK6sHHz";
+
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability("browserName", "chrome");
+        capabilities.setCapability("browserVersion", "123.0");
+        HashMap<String, Object> ltOptions = new HashMap<String, Object>();
+        ltOptions.put("username", "a23ska");
+        ltOptions.put("accessKey", "51pxhdIT5SNNtUNX3dbZAeYq5wfT2JMoAoPKTS1MguEsK6sHHz");
+        ltOptions.put("project", "Koel Automation");
+        ltOptions.put("selenium_version", "4.0.0");
+        ltOptions.put("w3c", true);
+        capabilities.setCapability("LT:Options", ltOptions);
+
+        return driver= new RemoteWebDriver(new URL(hubURL), capabilities);
     }
+    public void clickLoginBtn() {
+        WebElement submit = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[type='submit']")));
+        submit.click();
+    }
+
+    public void providePassword(String password) {
+        WebElement passwordField = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='password']")));
+        passwordField.sendKeys(password);
+    }
+
+    public void provideEmail(String email) {
+        WebElement emailField = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='email']")));
+        emailField.sendKeys(email);
+    }
+
+    public void navigateToPage() {
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+    }
+
+
+
+
+
+    public String getNotificationText() {
+        WebElement notificationT = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("div.success.show")));
+        return notificationT.getText();
+    }
+
+
+
+
+
+
+  //Mouse Hover
+
+
+    public void clickDeletePlaylistBtn() {
+        WebElement deletePlaylistBtn = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button.del.btn-delete-playlist")));
+        deletePlaylistBtn.click();
+    }
+
+    public void clickAnyPlaylist() {
+        WebElement clickPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)")));
+        clickPlaylist.click();
+    }
+
+
+
+
+
+    public String getRenamePlaylistSuccessMsg(){
+   WebElement notification = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+   return notification.getText();
```

Comment:
Page Object Model (POM): recommended to associate the methods with the corresponding page class where they are relevant. In the POM, each page or section of a web application is represented by a separate class. These classes encapsulate the elements and actions specific to that page or section.

Move these methods to the corresponding class according to the POM.

For example
![Screenshot_2](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/b07cd7c1-7977-4e53-80ea-8045ac17f779)